demos/browser/server.js

```diff
@@ -206,7 +217,59 @@ function serve(host = '127.0.0.1:8080') {
           console.warn("Cloud media capture not available")
           respond(response, 500, 'application/json', JSON.stringify({}))
         }
-      } else if (request.method === 'POST' && requestUrl.pathname === '/deleteAttendee') {
+      }
+      else if (request.method === 'POST' && requestUrl.pathname === '/startLiveConnector') {
+        if (ivsEndpoint) {
+          const callerInfo = await sts.getCallerIdentity().promise()
+          liveConnectorPipelineInfo = await chimeRegional.createMediaLiveConnectorPipeline({
+            Sinks : [
+              {
+                RTMPConfiguration : {
+                  AudioChannels : "Stereo",
+                  AudioSampleRate : "48000",
+                  Url: ivsEndpoint
+                },
+                SinkType: "RTMP"
+              }
+            ],
+            Sources : [
+              {
+                ChimeSdkMeetingLiveConnectorConfiguration : {
+                  Arn : `arn:aws:chime::${callerInfo.Account}:meeting:${meetingTable[requestUrl.query.title].Meeting.MeetingId}`,
+                  CompositedVideo: {
+                    GridViewConfiguration : {
+                      ContentShareLayout : "Vertical",
+                    },
+                    Layout: "GridView",
+                    Resolution: "FHD",
+                  },
+                  MuxType : "AudioWithCompositedVideo"
+                },
+                SourceType : "ChimeSdkMeeting"
+              }
+            ]
+          }).promise();
+          meetingTable[requestUrl.query.title].LiveConnector = liveConnectorPipelineInfo.MediaLiveConnectorPipeline;
+          respond(response, 201, 'application/json', JSON.stringify(liveConnectorPipelineInfo));
+        } else {
+          console.warn("Live Connector not available")
+          respond(response, 500, 'application/json', JSON.stringify({}))
+        }
+      }
+      else if (request.method === 'POST' && requestUrl.pathname === '/endLiveConnector') {
+        if (ivsEndpoint) {
+          liveConnectorPipelineId = meetingTable[requestUrl.query.title].LiveConnector.MediaPipelineId;
+          liveConnectorPipelineInfo = await chimeRegional.deleteMediaPipeline({
```

Comment:
For check: `endLiveConnector` should end the streaming or delete the media pipeline itself?

Reply:
Yes.